### DISCUSSION TITLE
--LogFilePath "LOG/SERVERNAME/{DatabaseName}/"

### DISCUSSION THREAD

**Discussion by soedro** [trimble-oss/sql-log-shipping-service#95] [Q&A]
Hello,

I am trying to make the service run but no luck.

I have installed the service, and I am trying to configure it. 
If I try to run it I am getting an error saying "app not responded in timely manner".

I have initialized the db on the destination server, it is in recovery mode.
Log files that will be used for restoring are in BLOB storage, location with SAS token is added to the Appsetings.json config file.

However, it looks like that I am having an issue with the Database Token you are mentioning in the instructions. What exactly is it? Searched for it but nothing came up.

--LogFilePath "LOG/SERVERNAME/{DatabaseName}/"  The {DatabaseName} token in the LogFilePath parameter will be replaced with the name of the database.

When I try to the command above this is the Error I am getting. 
Btw, there is no D: drive that I am using on this system.

2024-05-08 09:46:36.152 -04:00 [ERR] Error parsing command line options <1>
System.ArgumentException: Missing {DatabaseName} token from LogFilePath
   at LogShippingService.Config.set_LogFilePath(String value) in D:\a\sql-log-shipping-service\sql-log-shipping-service\sql-log-shipping-service\Config.cs:line 177
   at LogShippingService.Config.<>c__DisplayClass180_0.<ApplyCommandLineOptions>b__1(CommandLineOptions opts) in D:\a\sql-log-shipping-service\sql-log-shipping-service\sql-log-shipping-service\Config.cs:line 530 

Thanks!

**Comment by soedro** [comment]
I got it.

**Reply by DavidWiseman** [reply]
Hi, sorry I didn't see the notification for this discussion.  Glad you got it sorted.  The ContainerUrl and SASToken both need to be specified or it will default to disk file handling instead of azure blob.
